src/apps/backend/modules/logger/internals/types.ts

```diff
@@ -15,4 +15,5 @@ export enum LoggerTransport {
   Rollbar = 'rollbar',
   Grafana = 'grafana',
   Papertrail = 'papertrail',
+  Datadog = 'datadog',
```

Comment:
Should we remove support for all other logger types such as Rollbar, Grafana, Papertrail etc to reflect one opiniated approach on monitoring / logging from boilerplate standpoint or would that be part of separate PR?

Reply:
yes, @jjalan i'll remove it in a separate PR. I just wanted to test the end-to-end flow from logging to alert in the MS team via datadog before removing other options.